storage/src/adb/any.rs

```diff
@@ -386,8 +386,8 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Array, H: CHasher> Any<E, K, V,
             .iter()
             .map(|op| Any::<E, _, _, _>::op_digest(hasher, op))
             .collect::<Vec<_>>();
-
-        proof.verify_range_inclusion(hasher, &digests, start_pos, end_pos, root_hash)
+        let mut elements_iter = digests.iter().map(|d| d.as_ref());
```

Comment:
Now that you are accepting an iterator, you should be able to use `digests` directly.